src/main/java/net/sf/jsqlparser/util/deparser/SelectDeParser.java

```diff
@@ -334,6 +331,13 @@ public void visit(PlainSelect plainSelect) {
 
     }
 
+    protected void acceptWhere(PlainSelect plainSelect) {
```

Comment:
should probably be moved into the abstract class instead of re-implementing it several times.